docs/source/dialogue.rst

```diff
@@ -8,6 +8,71 @@ The figure below shows the dialogue flow in our system.
+-------
+
+User intents
+"""""""""""""
+
+:py:class:`UserIntents <moviebot.intents.user_intents>`
+
++------------------------+------------+
+| Intent   | Description   |
++========================+============+
+| Reveal   | The user wants to reveal a preference.   | 
++------------------------+------------+
+| Inquire  | Once the agent has recommended an item, the user can ask further details about it.        | 
++------------------------+------------+
+| Remove preference  | The user wants to remove any previously stated preference.        | 
++------------------------+------------+
+| Reject  | The user either has already seen/consumed the recommended item or does not like it.        | 
++------------------------+------------+
+| Accept  | The user accepts (likes) the recommendation. This will determine the success of the system as being able to find a recommendation the user liked.        | 
++------------------------+------------+
+| Continue recommendation  | If the user likes a recommendation, they can either restart, quit or continue the process to get a similar recommendation.        | 
++------------------------+------------+
+| Restart  | The user wants to restart the recommendation process.        | 
++------------------------+------------+
+| Acknowledge  | Acknowledge the agent's question where required.        | 
++------------------------+------------+
+| Deny  | Negate the agent's question where required.        | 
++------------------------+------------+
+| Hi  | When the user initiates the conversation, they start with a formal hi/hello or reveal preferences.        | 
++------------------------+------------+
+| Bye  | End the conversation by sending a bye message or an exit command.        | 
++------------------------+------------+
+
+Agent intents
+"""""""""""""
+
+:py:class:`AgentIntents <moviebot.intents.agent_intents>`
+
++------------------------+------------+
+| Intent   | Description   |
++========================+============+
+| Elicit   | Ask the user to describe their preferences.   | 
++------------------------+------------+
+| Recommend  | Based on the user's preferences, make a recommendation.        | 
++------------------------+------------+
+| No results  | The database does not contain any items matching the user's preferences.        | 
++------------------------+------------+
+|  Count results | The number of items matching the user's preferences is larger than a maximum limit. This will be followed by an elicit intent.        | 
++------------------------+------------+
+| Inform  | If the user inquires about the recommended item, the agent provides the relevant information.        | 
++------------------------+------------+
+| Continue recommendation  | If the user likes a recommendation, they can either restart, quit or continue the process to get a similar recommendation.        | 
++------------------------+------------+
+| Restart  | The user wants to restart the recommendation process.        | 
```

Comment:
Description should focus on agents intent. Not sure it is actually used, as the conversation simply restarts.